pyscraper/sp/parse-official-reports-new.py

```diff
@@ -665,6 +704,9 @@ def parse_html(session, report_date, soup, page_id, original_url):
                         current_votes.add_vote(current_division_way, tidied_paragraph, member_vote)
                     elif maybe_time:
                         current_time = maybe_time
+                    elif speech_part == "\n":
```

Comment:
Idly wondering, could this go up to line 671ish to save calling all the code on divisions/time/votes on a pointless string? Perhaps checking if `tidied_paragraph == ''` as tidy_string will have stripped it?